source/ui/dataman/AMScanView.h

```diff
@@ -323,26 +346,17 @@ class AMScanViewMultiView : public AMScanViewInternal {
 
 public:
 	explicit AMScanViewMultiView(AMScanView* masterView);
-
 	virtual ~AMScanViewMultiView();
 
 public slots:
-
-	/// Sets the visibility of the plot item cursor.
-	void setPlotCursorVisibility(bool visible);
-	/// Sets the coordinates for the plot item cursor.
-	void setPlotCursorCoordinates(const QPointF &coordinates);
-	/// Overloaded.  Sets the x-axis coordinate only.
-	void setPlotCursorCoordinates(double xCoordinate);
-	/// Sets the color of the plot cursor.
-	void setPlotCursorColor(const QColor &color);
-
```

Comment:
I've reimplemented these methods for AMScanView, which should allow other applications to maintain their current functionality.